google/resource_container_cluster.go

```diff
@@ -476,13 +587,23 @@ func resourceContainerClusterRead(d *schema.ResourceData, meta interface{}) erro
 
 	masterAuth := []map[string]interface{}{
 		{
-			"username":               cluster.MasterAuth.Username,
-			"password":               cluster.MasterAuth.Password,
-			"client_certificate":     cluster.MasterAuth.ClientCertificate,
-			"client_key":             cluster.MasterAuth.ClientKey,
-			"cluster_ca_certificate": cluster.MasterAuth.ClusterCaCertificate,
+			"username":                 cluster.MasterAuth.Username,
+			"password":                 cluster.MasterAuth.Password,
+			"client_certificate":       cluster.MasterAuth.ClientCertificate,
+			"client_key":               cluster.MasterAuth.ClientKey,
+			"cluster_ca_certificate":   cluster.MasterAuth.ClusterCaCertificate,
+			"issue_client_certificate": true,
```

Comment:
Instead of setting it to true here and potentially overriding it later, how about just set it later?

Reply:
Not a problem. Default behavior is "true". This is a flag vs an actual setting so I added code so that it behaves "as a setting" by checking for the existence of the client certitiface